### PR TITLE
[SS-4229] - Sidecar - Update handlebar to latest to address security …

### PR DESCRIPTION
**Pull request by sidharthjonnala-sugarcrm**
…vulnerabilities

Before creating a pull-request, please check https://github.com/wycats/handlebars.js/blob/master/CONTRIBUTING.md first.

Generally we like to see pull requests that

- [ ] Maintain the existing code style
- [ ] Are focused on a single change (i.e. avoid large refactoring or style adjustments in untouched code if not the primary goal of the pull request)
- [ ] Have good commit messages
- [ ] Have tests
- [ ] Don't significantly decrease the current code coverage (see coverage/lcov-report/index.html)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "handlebars",
   "barename": "handlebars",
-  "version": "1.3.3-sugarcrm-temporary",
+  "version": "1.3.4-sugarcrm-temporary",
```
Comment (sidharthjonnala-sugarcrm) [inline package.json]: This is temporary for testing purposes. Actual version will be done after QA. Version update needs to be done here and in other places where it references to the old version

**Review by ewahlquist-sugarcrm — APPROVED** [review]
Are the fixes in this PR borrowed from the code of a newer version of Handlebars, or are they our own creation? If it's the former, it might help to link to where the code came from so we have a reference to it

**Comment by sidharthjonnala-sugarcrm** [comment]
That would be a question for @mgasper-sugarcrm but Eric I think it will definitely be helpful to document the answer to that question on the associated Jira ticket

**Comment by mgasper-sugarcrm** [comment]
@ewahlquist-sugarcrm I went through all the issues that were attached on the task, and manually borrowed the code from the proposed fix commits to our codebase. The code of the fixes were from newer versions of handlebars.

**Review by kmelkumyan-sugarcrm — APPROVED** [review]

**Pull request merged**